### PR TITLE
Adding inference for MDC data

### PR DESCRIPTION
**Pull request by alecgunny**
- `mdc` project that reimplements most of the `infer` project, but for the datasets built by the MDC
- adding `deploy-mdc` to the pinto config, with more or less same arguments
- updating train/test start/stop times to match MDC segment times

Need to
- [x] Run and test that it works
- [x] Document the args passed to the MDC data generation script to build the necessary test dataset
- [ ] Possibly add `time_var` (the number of seconds of leeway you have to detect an event in the MDC) as a parameter, though just using 1 for now is probably sufficient.

I don't factor the `psd_length` into event detection at the moment, since this doesn't seem to cause loud background events and the main issue will just be removing entries too close to the edge from the `injections.hdf` file generated by the MDC.

### CODE REVIEW

File: projects/sandbox/mdc/mdc/main.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            Corresponds to the sample rate of the
+            output timeseries.
+        batch_size:
+            The number of subsequent windows to
+            include in a single batch of inference.
+        window_length:
+            Length of the window over which network
+            outputs should be locally integrated,
+            specified in seconds.
+        max_shift:
```
Comment (wbenoit26) [inline projects/sandbox/mdc/mdc/main.py:72]: Doesn't matter, but this makes me realize we need to change this in the `infer` project

File: projects/sandbox/mdc/mdc/main.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    with h5py.File(data_dir / "background.hdf", "r") as f:
+        size = len(f[ifos[0]][str(start)])
+    duration = size / sample_rate
+    end = start + duration
+
+    logging.info(f"Connecting to server at {ip}:8001")
+    client = InferenceClient(
+        f"{ip}:8001", model_name, model_version, callback=callback
+    )
+    chunk_size = int(chunk_size * sample_rate)
```
Comment (wbenoit26) [inline projects/sandbox/mdc/mdc/main.py:119]: So `chunk_size` is given in seconds? In `infer`, it looks like it refers to a number of indices

Comment (wbenoit26) [reply projects/sandbox/mdc/mdc/main.py:119]: Ah, nvm, I see where that conversion happens now

**Review by wbenoit26 — APPROVED** [review]
I tried to do a pretty thorough comparison against the files in the `infer` project, and there's nothing that jumps out to me as being wrong.

File: projects/sandbox/mdc/mdc/main.py (hunk 1 above)
Comment (alecgunny) [reply projects/sandbox/mdc/mdc/main.py:72]: Ah yeah good catch, we'll need to do a thorough doc review before publication where we weed out stuff like this

**Pull request merged**